### ISSUE TITLE
Unknown command 'app-placement'

### ISSUE THREAD

**Issue by drnic** [cloudfoundry-attic/tools-cf-plugin#3]
`cf app-placement` is mentioned in the README; but isn't being discovered:

```
$ cf app-placement
Unknown command 'app-placement'. See 'help' for available commands.
```

**Comment by mark-rushakoff** [comment]
It looks like the [gem version on Rubygems is still 2.4.0](http://rubygems.org/gems/tools-cf-plugin) but app-placement wasn't available until the 3.0.0 version bump.  @vito can you bump the version on Rubygems?

@mark-rushakoff / @matthewmcnew

**Comment by vito** [comment]
done

http://rubygems.org/gems/tools-cf-plugin/versions/3.0.0
